etcd-manager/pkg/volumes/scaleway/discovery.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package scaleway
+
+import (
+	"fmt"
+
+	"github.com/scaleway/scaleway-sdk-go/api/instance/v1"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/etcdadm/etcd-manager/pkg/privateapi/discovery"
+)
+
+// ScwVolumes also allows the discovery of peer nodes
```

Comment:
Go style suggests just calling this `Volumes`, it will be named `scaleway.Volumes` when imported outside of the package.  See "Avoid repetition" https://go.dev/blog/package-names

Just a nit / FYI though - not a blocker for merge.